components/datatypes/application.schema.json

```diff
@@ -29,6 +29,11 @@
           "type": "string",
           "description": "Version of the application."
         },
+        "xdm:sdkVersion": {
```

Comment:
Which sdk is this? An application could use many sdk's in its implementation. If this is intended to be the airship SDK version, then it seems like that should be in an airship specific field.